components/eamxx/tests/single-process/ace/input.yaml

```diff
@@ -0,0 +1,70 @@
+%YAML 1.1
+---
+driver_options:
+  atmosphere_dag_verbosity_level: 5
+
+time_stepping:
+  time_step: ${ATM_TIME_STEP}
+  run_t0: ${RUN_T0}
+  number_of_steps: ${NUM_STEPS}
+
+atmosphere_processes:
+  atm_procs_list: [ace]
+  ace:
+    checkpoint_path: /pscratch/sd/m/mahf708/emulator/ace_ckpt_traced${GPU_OR_CPU}.tar
```

Comment:
Let's migrate data to project-wide folders and rm all user-specific paths before merging.